### PR TITLE
feat(signup): experiment with larger unbranded sso buttons

### PR DESCRIPTION
**Pull request by raquelmsmith**
## Problem

Our current SSO buttons look nice because they are on-brand (ie. blue) but that means they might be less noticeable to people who are signing up.

If we look at the data, we can see that if someone clicks one of the SSO buttons, they have a [~50% chance](https://app.posthog.com/insights/avi7go0J) (internal only, make sure to view from step 2 to step 4) of completing signup. If someone starts filling out the form (puts their email address in) then they are [only ~35% likely](https://app.posthog.com/insights/MeL85GxH) (internal only, make sure to view from step 2 to step 3) to complete signup. So, we're definitely benefited by pushing people toward SSO login. @timgl wanted to try making the SSO buttons larger and more visible, so here we go!

## Changes

Behind the test feature flag, the social login buttons are now a bit larger and use icons that are consistent with the brand for the provider (eg. the Google G logo has the red/green/yellow/blue colors, it's not just our primary blue color). I've also moved them to the first login option, above the email/pw form, to make it seem like the default option.

| Before  | After |
| ------------- | ------------- |
| image | image  |

Another small change included here is sorting the SSO providers so that Google is first, since it's our most used.

## Measuring results

We'll run an experiment with these two variants (before and after, aka control and test) and measure:

1. The percentage of people who click the social login button vs start filling out the form
2. The conversion rate for signups overall

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?

No functionality changes, so just with my eyeballs.

### CODE REVIEW

**Comment by raquelmsmith** [comment]
I feel like @corywatilo will give me some pretty good side eye for this but I say we try it and see what happens.

File: frontend/src/scenes/authentication/signup/signupForm/SignupForm.tsx [hunk 1]
```diff
@@ -49,17 +48,6 @@ export function SignupForm(): JSX.Element | null {
             {panel === 0 ? (
                 <>
```
Comment (kappa90) [inline frontend/src/scenes/authentication/signup/signupForm/SignupForm.tsx:49]: We can remove these then

**Review by kappa90 — APPROVED** [review]
Left a comment, but good for me

**Comment by timgl** [comment]
To make the test possibly more impactful, should we put these logos above the email form?

**Pull request merged**